Day-5-Exercises/main.js

```diff
@@ -0,0 +1,217 @@
+const countries = [
+    'Albania',
+    'Bolivia',
+    'Canada',
+    'Denmark',
+    'Ethiopia',
+    'Finland',
+    'Germany',
+    'Hungary',
+    'Ireland',
+    'Japan',
+    'Kenya'
+  ]
+  
+  const webTechs = [
+    'HTML',
+    'CSS',
+    'JavaScript',
+    'React',
+    'Redux',
+    'Node',
+    'MongoDB'
+  ]
+
+//Exercise: Level 1
+
+// Declare an empty array;
+const arrEmpty = []
+
+// Declare an array with more than 5 number of elements
+const arr = [4, 57, 30, -1, 444, 9.99]
+console.log(arr)
+
+// Find the length of your array
+console.log(arr.length)
+
+// Get the first item, the middle item and the last item of the array
+console.log(arr[0], `${arr.length % 2?arr[Math.floor(arr.length / 2)]:'No middle element'}`, arr[arr.length -1] )
+
+// Declare an array called mixedDataTypes, put different data types in the array and find the length of the array. The array size should be greater than 5
+const mixedDataTypes = [45, 'Text', 8.44, true, {firstName: 'Dmitry', lastName: 'Lazarev'}, arr ]
+console.log(mixedDataTypes)
+
+// Declare an array variable name itCompanies and assign initial values Facebook, Google, Microsoft, Apple, IBM, Oracle and Amazon
+const itCompanies = ['Facebook', 'Google', 'Microsoft', 'Apple', 'IBM', 'Oracle', 'Amazon'] //
+
+// Print the array using console.log()
+console.log(itCompanies)
+
+// Print the number of companies in the array
+console.log('The are', itCompanies.length, 'IT companies')
+
+// Print the first company, middle and last company
+console.log(itCompanies[0], itCompanies[Math.floor(itCompanies.length / 2)], itCompanies[itCompanies.length - 1])
+
+// Print out each company
+itCompanies.forEach(element => {
+    console.log(element)
+});
+
+// Change each company name to uppercase one by one and print them out
+itCompanies.forEach((element,index) => {
+    //if changes needs to be applyed to initial array
+    //itCompanies[index] = element.toUpperCase() 
+    //console.log(itCompanies[index])
+    
+    //if only need to print out in Upper case
+    console.log(element.toUpperCase()) 
+});
+
+// Print the array like as a sentence: Facebook, Google, Microsoft, Apple, IBM,Oracle and Amazon are big IT companies.
+console.log(itCompanies.join(', '))
+
+// Check if a certain company exists in the itCompanies array. If it exist return the company else return a company is not found
+let someCompany = 'IBM'
+console.log(`${itCompanies.includes(someCompany)?someCompany:'Company is not found'}`)
+
+// Filter out companies which have more than one 'o' without the filter method
+const tempArr = []
+itCompanies.forEach((element, index) => {    
+    if (element.split('o').length <= 2) {tempArr.push(element)};   
```

Comment:
You should use the greater than or equal operator to find at least if the length is equal to 2.

Reply:
As agreed - will leave this solution as correct one